### PR TITLE
FB-712 (Error handling)

### PR DESCRIPTION
**Pull request by sequencemedialimited**
[FB-712](https://dsdmoj.atlassian.net/browse/FB-712)

- Reopened to account for error code 422

### CODE REVIEW

File: create.html [hunk 1]
```diff
@@ -1,313 +1,344 @@
...
+    <script src="assets/govuk-frontend-design-system.min.js"></script>
+    <script>
+    const { execSync } = require('child_process')
+    const { app } = require('./app.js')
+    const { git } = app
+    const { pathExists, isDirectory, glob, request } = app.utils
+    const path = require('path')
+
+    function createErrorMessage(e, serviceName) {
+        let errorMessage
```
Comment (asmega) [inline create.html]: is your convention to use 4 spaces? i think we use 2 everywhere else

Comment (sequencemedialimited) [reply create.html]: I added an HTML prettifier into Sublime, and this is how it does JS :(

I'll add the `eslint-plugin-html` and do it again :)

Comment (sequencemedialimited) [reply create.html]: I've left the HTML as the prettifier did it, but ESLint now has visibility of the JS in those files

Comment (sequencemedialimited) [reply create.html]: (The CLI tool "Prettier" smooshes the JS, even with ESLint applied after. [FB-711](https://dsdmoj.atlassian.net/browse/FB-711) should take care of these things properly)

**Review by emileswarts — APPROVED** [review]

**Pull request merged**